.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 
-matrix:
```

Comment:
The `matrix` keyword is deprecated: https://docs.travis-ci.com/user/build-matrix/